examples/sample-app/application-template-dockerbuild.json

```diff
@@ -83,14 +83,14 @@
       "kind": "ImageStream",
       "apiVersion": "v1",
       "metadata": {
-        "name": "ruby-27-centos7"
+        "name": "ruby-27"
       },
       "spec": {
         "tags": [
           {
             "from": {
               "kind": "DockerImage",
-              "name": "docker.io/centos/ruby-27-centos7:latest"
+              "name": "registry.access.redhat.com/ubi8/ruby-27:latest"
```

Comment:
shouldn't we use `registry.redhat.io`? Clusters should always have a valid pull secret.

Reply:
What about OKD?  When we moved "centos" (OKD) imagestreams to UBI, I was told explicitly to use that registry since they *won't* necessarily have a registry.redhat.io auth in their pull secret.

Reply:
I had not considered OKD - then yes that won't have a pull secret by design.